images/node/Dockerfile-debian

```diff
@@ -0,0 +1,38 @@
+###############################################################
+# DOCKER IMAGE - Debian based Nodejs Application (Express.js) #
+###############################################################
+
+FROM node:20-bookworm-slim
+
+# Set the working directory within the container
+WORKDIR /app
+
+# Copy both package.json and package-lock.json
+COPY package*.json ./
+
+# Install application dependencies
+RUN npm install
+
+# Copy the application code into the container
+COPY . /app
+
+# Omitting dev dependencies
+RUN npm ci --omit=dev
```

Comment:
This line should be removed. We should install dependencies on line 14 using the command "RUN npm ci --omit=dev"

npm ci is another way of installing dependencies that is more reliable (google it - its interesting)